test/csuite/wt4105_large_doc_small_upd/main.c

```diff
@@ -133,7 +133,7 @@ main(int argc, char *argv[])
             modify_entry.offset = offset;
             modify_entry.size = modify_entry.data.size;
             /* FIXME-PM-1521: extend timeout to pass the test */
-            (void)alarm(5);
+            (void)alarm(7);
```

Comment:
Not sure why overhead increases and I have to extend the timeout further to pass this test.